### DISCUSSION TITLE
We shall introduce a file Structure in the readme file.

### DISCUSSION THREAD

**Discussion by ranbir7** [Autostronomy/AstroPhot#40] [Ideas]
Here we write which Directory has which type of files ,
like their(<strong>each directory's or sub-directory's</strong>) main purpose towards this project.
Making it better for people to navigate the Directories easily.

**Comment by ConnorStoneAstro** [comment]
For anyone who just does `pip install autoprof` they don't really need to know the file structure. But I think that is a great idea to help contributors! We could include something like that in the `CONTRIBUTING.md` document and I bet it would be very helpful! Is there other information that you think would get contributors started more easily?

**Reply by ranbir7** [reply]
okay 
I'll find solutions to it and tell u soon as I reach conclusion👍
